.gitignore

```diff
@@ -1,3 +1,4 @@
 release
 coverage.out
 vendor
+.idea
```

Comment:
We are not adding any IDE-specific ignores, please add it to your local global gitignore.